setup.cfg

```diff
@@ -51,35 +51,40 @@ keywords =
 
 [options]
 zip_safe = True
-install_requires = 
+install_requires =
+    appdirs ~= 1.4
 	depinfo
+    diskcache ~= 5.0
 	future
+    httpx ~= 0.14
 	importlib_resources
 	numpy ~= 1.13
 	optlang ~= 1.4
 	pandas ~= 1.0
+    pydantic ~= 1.6
 	python-libsbml-experimental == 5.18.1
+    rich ~= 6.0
```

Comment:
If we depend on rich we should probably update our logging to use that. Can be a different PR though.

Reply:
Nice alternative is tqdm but rich indeed opens the door to many console improvements.

Reply:
I think rich is fine since it bundles a lot of console stuff. We don't have that many progress bars that we need tqdm.